addons/website/static/src/xml/website.backend.xml

```diff
@@ -6,33 +6,37 @@
     </div>
 
     <t t-name="website.WebsiteDashboardMain">
-        <div class="container o_website_dashboard">
+        <div class="dashboards">
+            <div class="container-fluid o_website_dashboard">
+            </div>
         </div>
     </t>
 
     <t t-name="website.dashboard_visits">
-        <div class="row o_dashboard_visits">
+        <div class="o_dashboard_visits">
             <div class="col-xs-12 o_box">
-                <h2>Visits</h2>
-
-                <div t-if="widget.dashboards_data.visits &amp;&amp; widget.dashboards_data.visits.ga_client_id" class="row js_analytics_components">
+                
+                <div t-if="widget.dashboards_data.visits &amp;&amp; widget.dashboards_data.visits.ga_client_id" class="js_analytics_components">
                 </div>
 
-                <div t-if="!(widget.dashboards_data.visits &amp;&amp; widget.dashboards_data.visits.ga_client_id)">
+                <div class="col-md-12" t-if="!(widget.dashboards_data.visits &amp;&amp; widget.dashboards_data.visits.ga_client_id)">
+                    <h2>Visits</h2>
                     <div class="o_demo_background">
                         <div class="o_layer">
                         </div>
-                        <button class="btn btn-sm btn-primary js_link_analytics_settings">Link my Analytics Account</button>
+                        <div class="o_buttons text-center">
+                            <h3>Currently there is no data available.</h3>
+                            <button class="btn btn-sm btn-primary js_link_analytics_settings center-block mb8">Link my Analytics Account</button>
+                            <a href="#" class="js_link_analytics_settings">Edit my Analytics Client ID</a>
```

Comment:
If you do that you will not be able to edit your client id after the first link, which is the only purpose of this link.